src/Aggregation.ts

```diff
@@ -0,0 +1,54 @@
+/** AggreationType will be used to create type of aggreation when querying history states.
+  <ul>
+  <li>AggregationType.COUNT: count function.</li>
+  <li>AggregationType.SUM: sum function. </li>
+  <li>AggregationType.MAX: max function.</li>
+  <li>AggregationType.MIN: min function.</li>
+  <li>AggregationType.MEAN: mean function.</li>
+  </ul>
+*/
+export const FunctionType = {
+    COUNT: "COUNT",
+    SUM: "SUM",
```

Comment:
I found string literal.
http://qiita.com/ConquestArrow/items/a2a657ce19feaf12f82f

What do yo think about using string literal instead of dictionary?